packages/vim/build.sh

```diff
@@ -6,9 +6,9 @@ TERMUX_PKG_DEPENDS="libiconv, ncurses, vim-runtime"
 TERMUX_PKG_RECOMMENDS="diffutils"
 # vim should only be updated every 50 releases on multiples of 50.
 # Update both vim and vim-python to the same version in one PR.
```

Comment:
vim-python must be updated too, and this is not a multiple of 50.

Reply:
Ah, my mistake.